app/src/androidTest/java/com/github/onlynotesswent/ui/search/SearchTest.kt

```diff
@@ -25,26 +24,29 @@ class SearchScreenTest {
 
   @get:Rule val composeTestRule = createComposeRule()
 
-  private val mockNotes =
+  private val testNote1 = Note(
+      id = "",
```

Comment:
Maybe for consistency is better to put an id that is not empty

Reply:
Yes, sorry I didn't see it, I was just refactoring the previous list into variables to improve modularity. Next time someone makes a PR on the Search feature we will in any case need to overhaul the tests and fix this.